src/components/Card.css

```diff
@@ -3,24 +3,74 @@
     height: 150px;
     position: relative;
     border: none;
-    padding: 0;
+    padding: 25px;
     background: transparent;
 }
 
 .Card[data-color="red"] {
     background: red;
+    margin: 5px;
+}
+
+.Card.almighty {
+    background-image: url('../assets/map_cards/Red/almighty.webp');
+    background-size: 100% 100%;
```

Comment:
I was hastily writing this out during our mini tutorial session, so I put this rule in the wrong place. If you move it to the main `.Card` selector, you can delete it from all of the individual card variants. Then, to kinda condense similar things a bit, you could change all of the individual rules to look like this:
```css
/* -- RED CARDS -- */
.Card.almighty   { background-image: url('../assets/map_cards/Red/almighty.webp'); }
.Card.blackroom  { background-image: url('../assets/map_cards/Red/blackroom.webp'); }
.Card.bottomless { background-image: url('../assets/map_cards/Red/bottomless.webp'); }
/* ... etc. ... */

/* -- GREEN CARDS -- */
.Card.daze { /* you get the point */ }
```
Also nice touch keeping things sorted alphabetically! I was gonna request that if you didn't. 😄

Reply:
To be clear, I mean the rule on L17, `background-size: 100% 100%;`, is the one that should be moved out to `.Card`. I thought it would highlight that line specifically.